docs/framework/docker/aspnetmvc.md

```diff
@@ -50,43 +45,39 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
-[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/)
-or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server). 
+The development machine must be running
+[Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/) (or higher)
+or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server) (or higher). 
 
-Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/),
-version 1.12 Beta 26, or newer. Windows Container support is only available
-in the Beta channel at this time.
+[Docker for Windows](https://docs.docker.com/docker-for-windows/) - version Stable 1.13.0 or 1.12 Beta 26 (or newer versions)
 
 > [!IMPORTANT]
-> If you are using Windows Server 2016, you need to follow the
-> instructions for [Container Host Deployment - Windows Server](https://msdn.microsoft.com/virtualization/windowscontainers/deployment/deployment)
-> before you can run Docker containers.
+> If you are using Windows Server 2016, follow the
+> instructions for [Container Host Deployment - Windows Server](https://msdn.microsoft.com/virtualization/windowscontainers/deployment/deployment).
 
-After installing and starting Docker, you'll need to right-click on the
-tray icon and select **Switch to Windows containers** in order to run
+After installing and starting Docker,  right-click on the
```

Comment:
Why present Docker as a prereq and then say "after installing"?  Maybe better to say install it when the link is presented earlier.